<h3>Exception Generated On 13 नवम्बर 2021 @ 20:14</h3><br /><br /><strong>Page Location:</strong> /api/address/add<br /><br /><strong>Client IP Address:</strong> 127.0.0.1<br /><br /><strong>Error Message:</strong> Error Adding Brand  : 
System.Data.Entity.Infrastructure.DbUpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.Entity.Core.UpdateException: An error occurred while updating the entries. See the inner exception for details. ---> System.Data.SqlClient.SqlException: Cannot insert the value NULL into column 'AddressOfUserId', table 'xog_db.dbo.Addresses'; column does not allow nulls. INSERT fails.
The statement has been terminated.
   at System.Data.SqlClient.SqlCommand.<>c.<ExecuteDbDataReaderAsync>b__180_0(Task`1 result)
   at System.Threading.Tasks.ContinuationResultTaskFromResultTask`2.InnerInvoke()
   at System.Threading.Tasks.Task.Execute()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Data.Entity.Utilities.TaskExtensions.CultureAwaiter`1.GetResult()
   at System.Data.Entity.Core.Mapping.Update.Internal.DynamicUpdateCommand.<ExecuteAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.<UpdateAsync>d__0.MoveNext()
   --- End of inner exception stack trace ---
   at System.Data.Entity.Core.Mapping.Update.Internal.UpdateTranslator.<UpdateAsync>d__0.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Data.Entity.Core.Objects.ObjectContext.<ExecuteInTransactionAsync>d__3d`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Data.Entity.Core.Objects.ObjectContext.<SaveChangesToStoreAsync>d__39.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Data.Entity.SqlServer.DefaultSqlExecutionStrategy.<ExecuteAsyncImplementation>d__9`1.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Data.Entity.Core.Objects.ObjectContext.<SaveChangesInternalAsync>d__31.MoveNext()
   --- End of inner exception stack trace ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter`1.GetResult()
   at XOG.AppCode.BLL.AddressBL.<AddAsync>d__9.MoveNext() in D:\Projects\XOGProject\XOG.API\XOG.API\AppCode\BLL\AddressesBL.cs:line 176<br /><br /><strong>Error Details:</strong> <br /><br /><strong>Method Name:</strong> <br /><br /><strong>Source:</strong> <br /><br /><strong>Stack Trace:</strong><br />NA<br /><hr /><br /><h3>Mail Exception</h3><br /><br /><strong>Error Message:</strong> The SMTP host was not specified.<br /><br /><strong>Error Details:</strong> <br /><br /><strong>Method Name:</strong> CheckHostAndPort<br /><br /><strong>Source:</strong> System<br /><br /><strong>Stack Trace:</strong><br />   at System.Net.Mail.SmtpClient.CheckHostAndPort()
   at System.Net.Mail.SmtpClient.Send(MailMessage message)
   at XOG.Helpers.MailHelper.SendUsingSmtp(IdentityMessage message) in D:\Projects\XOGProject\XOG.API\XOG.API\AppCode\Helpers\MailHelper.cs:line 213
   at XOG.Helpers.MailHelper.Send(IdentityMessage message) in D:\Projects\XOGProject\XOG.API\XOG.API\AppCode\Helpers\MailHelper.cs:line 105
   at XOG.Util.ErrorLogger.LogError(Exception ex, DbEntityValidationException e) in D:\Projects\XOGProject\XOG.API\XOG.API\AppCode\Util\ErrorLogger.cs:line 24<hr /><br />
